### PR TITLE
Gene View Controller Polish

### PR DESCRIPTION
**Pull request by orta**
* Adds an Eigen-like back nav ( it doesn't go away on scroll, unlike eigen's ) is that feature deprecated? I'm not seeing it in any RN component's API

* Tightens up the UI, so it looks like most of the screenshots now, with normal margins/padding etc

* The header / sticky bits is still definitely a WIP, I've tried to show off how it feels in the GIF

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG added.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/ecd781c1e0baa75451567bc45c6615d3cfb929fe/lib/components/artwork_grids/infinite_scroll_grid.js">lib/components/artwork_grids/infinite_scroll_grid.js</a>, <a href="https://github.com/artsy/emission/blob/ecd781c1e0baa75451567bc45c6615d3cfb929fe/lib/components/buttons/flat_white.js">lib/components/buttons/flat_white.js</a>, <a href="https://github.com/artsy/emission/blob/ecd781c1e0baa75451567bc45c6615d3cfb929fe/lib/components/gene/about.js">lib/components/gene/about.js</a> &amp; <a href="https://github.com/artsy/emission/blob/ecd781c1e0baa75451567bc45c6615d3cfb929fe/lib/containers/gene.js">lib/containers/gene.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by alloy** [comment]
> Adds an Eigen-like back nav ( it doesn't go away on scroll, unlike eigen's ) is that feature deprecated? I'm not seeing it in any RN component's API

Nope, that’s left up to Eigen: https://github.com/artsy/eigen/blob/7a3949d768a5ffca05f21c68cf7f5f531411acad/Artsy/App/ARAppDelegate%2BEmission.m#L152-L198

It should maybe move to the base component VC, though.

File: lib/components/buttons/flat_white.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import { Animated, StyleSheet, TouchableHighlight, View } from 'react-native'
+
+import Headline from '../text/headline'
+import colors from '../../../data/colors'
+
+const AnimationDuration = 250
+const AnimatedTouchable = Animated.createAnimatedComponent(TouchableHighlight)
+const AnimatedHeadline = Animated.createAnimatedComponent(Headline)
+
+export default class InvertedButton extends React.Component {
```
Comment (alloy) [inline lib/components/buttons/flat_white.js]: Is it any different and/or should it be renamed?

**Comment by alloy** [comment]
Starting to look nice 👌

File: lib/components/buttons/flat_white.js (hunk 1 above)
Comment (orta) [inline lib/components/buttons/flat_white.js]: renamed, it's not got the intermediary disabled state

**Pull request merged**